crowbar_framework/app/models/node_object.rb

```diff
@@ -1351,7 +1351,7 @@ def link_to_device?(device, linkname)
     return true if device == linkname
     lookup_and_name = linkname.gsub(/^\/dev\/disk\//, '').split(File::SEPARATOR, 2)
     linked_devs = @node[:block_device][device][:disks][lookup_and_name[0]] rescue []
-    linked_devs.include?(lookup_and_name[1])
+    linked_devs.include?(lookup_and_name[1]) rescue false
```

Comment:
Our (ab)use of `rescue` is starting to become frightening :/ But I guess I'm fine with this for now :-)